var/spack/repos/builtin/packages/gobject-introspection/package.py

```diff
@@ -53,6 +53,13 @@ class GobjectIntrospection(Package):
     #   inserted into the scripts as they're generated.
     patch("sbang.patch")
 
+    # Drop deprecated xml.etree.ElementTree.Element.getchildren() which leads
+    # to compilation issues with Python 3.9.
+    # https://gitlab.gnome.org/GNOME/gobject-introspection/-/issues/325
+    patch('https://gitlab.gnome.org/GNOME/gobject-introspection/-/commit/'
+          '1f9284228092b2a7200e8a78bc0ea6702231c6db.patch',
+          sha256='7700828b638c85255c87fcc317ea7e9572ff443f65c86648796528885e5b4cea')
```

Comment:
According to https://gitlab.gnome.org/GNOME/gobject-introspection/-/commit/1f9284228092b2a7200e8a78bc0ea6702231c6db, this seems to be fixed starting in version 1.63.2, so we should probably add `when='@:1.63.1'`.

Reply:
Thanks @michaelkuhn Solved!